docker/gitlab-content-ci/Pipfile

```diff
@@ -0,0 +1,55 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
```

Comment:
Do we need all these packages? Don't we use the virtual environment in the `venv` directory for most python script executions?

Reply:
As discussed, this is done to eliminate the usage of build-requirements.txt file and the packages here are the ones needed for the build environment to run.